radixdlt-java/src/test/java/com/radixdlt/client/core/network/actions/FetchAtomsCancelActionTest.java

```diff
@@ -0,0 +1,18 @@
+package com.radixdlt.client.core.network.actions;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.mock;
+
+import com.radixdlt.client.atommodel.accounts.RadixAddress;
+import org.junit.Test;
+
+public class FetchAtomsCancelActionTest {
+	@Test
+	public void when_initializing_with_null__npe_should_be_thrown() {
```

Comment:
Did we discuss using `_is_thrown` and `_is_not_thrown` for these?

Reply:
We did, and we decided on _exception_is_thrown and _exception_is_not_thrown variants.